packaging/debian/preinst

```diff
@@ -17,15 +17,3 @@ if ! id cwagent >/dev/null 2>&1; then
      echo "create user cwagent, result: $?"
 fi
 
-if ! grep "^aoc:" /etc/group >/dev/null 2>&1; then
-     groupadd -r aoc >/dev/null 2>&1
-     echo "create group aoc, result: $?"
-fi
-
-if ! id aoc >/dev/null 2>&1; then
```

Comment:
It should be safe to remove this since we had triggered the roll-back and also there has not been any customers who report this issue.